modules/gdscript/gdscript_functions.cpp

```diff
@@ -1632,7 +1632,7 @@ MethodInfo GDScriptFunctions::get_info(Function p_func) {
 			return mi;
 		} break;
 		case MATH_SMOOTHSTEP: {
-			MethodInfo mi("smoothstep", PropertyInfo(Variant::FLOAT, "from"), PropertyInfo(Variant::FLOAT, "to"), PropertyInfo(Variant::FLOAT, "weight"));
+			MethodInfo mi("smoothstep", PropertyInfo(Variant::FLOAT, "from"), PropertyInfo(Variant::FLOAT, "to"), PropertyInfo(Variant::FLOAT, "x"));
```

Comment:
You're correct that weight is inaccurate, but also "x" isn't very descriptive here. Perhaps "between", since meaningful values exist on the range of "from" to "to" (aka: bewteen them)?

Reply:
It does not need to be between the two values, though. Values to the right and to the left are meaningful as well, even if they are clamped to the endpoint.

Reply:
Perhaps `value` or `s`, which while not too descriptive either is consistent with the other mathematical functions like `exp` or `sqrt` (smoothstep is a mathematical function too, after all).

Reply:
Well, those valures are valid in that you still get expected outputs, but it's not what I would describe as "meaningful".

Both of your proposals are better than "x" IMO. I'll let you make the final call, but I think "between" still makes sense.

Reply:
I would say it makes enough sense to have values outside (or just outside) the range too that being overly prescriptive with the naming might be inaccurate.
Another part of why I prefer `x` or `s` is that various other sources do this rather than naming it (wikipedia, opengl https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/smoothstep.xhtml, others, and I do not believe one would find the variable named between in these other sources).
I am leaning towards `between` potentially causing more confusion than it helps, for example since one often says that a value lies between other values, or that one interpolates between other values, and that it is not immediately clear that `between` refers to that the "betweened" value.